src/js/components/modals/AppModalComponent.jsx

```diff
@@ -61,28 +61,19 @@ var AppModalComponent = React.createClass({
     Mousetrap.bind(["command+enter", "ctrl+enter"], () => this.handleSubmit());
 
     if (!this.hasInfoObject()) {
-      InfoStore.on(InfoEvents.CHANGE, this.onInfoChange);
-      InfoStore.on(InfoEvents.REQUEST_ERROR, this.onInfoChange);
+      InfoStore.once(InfoEvents.CHANGE, this.onInfoChange);
```

Comment:
That's what `once` is for: it only attaches the listener once and automagically removes it afterwards. https://nodejs.org/api/events.html#events_handling_events_only_once .